client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/aggregation/RemoteClusteredAggregationExpiringTest.java

```diff
@@ -0,0 +1,129 @@
+import org.infinispan.commons.dataconversion.MediaType;
+import org.infinispan.commons.test.CommonsTestingUtil;
+import org.infinispan.commons.test.annotation.TestForIssue;
+import org.infinispan.commons.util.Util;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.StorageType;
+import org.infinispan.persistence.dummy.DummyInMemoryStoreConfigurationBuilder;
+import org.infinispan.protostream.SerializationContextInitializer;
+import org.infinispan.query.model.Task;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "client.hotrod.query.aggregation.RemoteClusteredAggregationExpiringTest")
+@TestForIssue(githubKey = "13194")
+public class RemoteClusteredAggregationExpiringTest extends MultiHotRodServersTest {
+
+   private final String indexDirectory = CommonsTestingUtil.tmpDirectory(getClass());
+
+   /*
+   <?xml version="1.0"?>
+   <distributed-cache name="TEST_TASKS" owners="1" mode="SYNC" remote-timeout="17500" statistics="true">
+      <encoding>
+         <key media-type="application/x-protostream"/>
+         <value media-type="application/x-protostream"/>
+      </encoding>
+      <locking concurrency-level="1000" acquire-timeout="15000" striping="false"/>
+      <expiration lifespan="-1"/>
+      <memory storage="HEAP" max-count="1000"/>
+      <persistence passivation="false">
+         <file-store purge="false" read-only="false"/>
+      </persistence>
+      <indexing enabled="true" storage="filesystem">
+         <indexed-entities>
+            <indexed-entity>test.Task</indexed-entity>
+         </indexed-entities>
+      </indexing>
+      <state-transfer timeout="60000"/>
+   </distributed-cache>
+   */
+   @Override
+   protected void createCacheManagers() throws Throwable {
+      Util.recursiveFileRemove(indexDirectory);
+      assertThat(new File(indexDirectory).mkdirs()).isTrue();
+
+      ConfigurationBuilder config = hotRodCacheConfiguration(getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC));
+      config.clustering()
+       .cacheMode(CacheMode.DIST_SYNC)
+            .remoteTimeout(17500)
+            .hash().numOwners(1);
+      config.statistics().enable();
+      config.encoding().key().mediaType(MediaType.APPLICATION_PROTOSTREAM);
+      config.encoding().value().mediaType(MediaType.APPLICATION_PROTOSTREAM);
+      config.locking().concurrencyLevel(1000).lockAcquisitionTimeout(15000);
+      config.expiration().lifespan(-1);
+      config.memory().storage(StorageType.HEAP).maxCount(1000);
+      config.persistence().passivation(false)
+            .addStore(new DummyInMemoryStoreConfigurationBuilder(config.persistence()));
+      config.indexing().enable()
+            .storage(FILESYSTEM).path(indexDirectory)
+            .addIndexedEntity("model.Task");
+      config.clustering().stateTransfer().timeout(60000);
```

Comment:
can we remove any unnecessary configuration? Most of them seem irrelevant and you only need indexing.

Reply:
I added them to copy the reproducer provided with the issue. But, sure, we can simplify